openquake/hazardlib/geo/surface/gridded.py

```diff
@@ -0,0 +1,164 @@
+from openquake.hazardlib.geo.surface.base import BaseSurface
+from openquake.hazardlib.geo.mesh import Mesh
+
+
```

Comment:
This looks like an Abstract Base Class. If that is the intent, why not defining it so? I assume you will add concrete implementations later on.